### PR TITLE
CASM-4673: Documentation for Kyverno signature validation policy

### PR DESCRIPTION
**Pull request by pradeepkumargl**
Kyverno failed to verify tls certificate for signature verification. Added workaround to the known issues section.

Testing Logs:

[background-disable.txt](https://github.com/user-attachments/files/17438886/background-disable.txt)
[restart_pods.txt](https://github.com/user-attachments/files/17438892/restart_pods.txt)

### CODE REVIEW

**Review by mtupitsyn — APPROVED** [review]

**Comment by mtupitsyn** [comment]
@nrockershousen Could you please take a look?

File: operations/kubernetes/Kyverno.md [hunk 1]
```diff
@@ -432,3 +432,14 @@ and [match/exclude](https://release-1-10-0.kyverno.io/docs/writing-policies/matc
...
+* Kyverno failed to verify `tls` certificate for signature verification from CSM 1.6 and above
+
+    ```bash
+    10m         Warning   PolicyViolation   pod/canu-test-6ff597df6d-fbnbd   policy check-image/check-image fail: failed to 
+    verify image registry.local/artifactory.algol60.net/csm-docker/stable/cray-canu/canu-test:1.6.36: 
+    .attestors[0].entries[0].keys: Get "https://registry.local/v2/": tls: failed to verify certificate: x509: certificate 
+    signed by unknown authority
+    ```
+
+    When end user encounters the above issue for signature verification then the solution is to restart the Kyverno pods.
```
Comment (mtupitsyn) [inline operations/kubernetes/Kyverno.md:444]: @pradeepkumargl I do not quite understand this solution ... Was it really tested?

From what I see, Platform TLS certificate is installed by Helm post-install hook. This hook is not executed when pods are simply restarted. It is only executed when Helm chart is installed or upgraded.

**Review by mtupitsyn — CHANGES_REQUESTED** [review]
Please correct me if I'm wrong - my understanding is that you need to re-deploy chart in order to re-run post-install Helm hook. Restating pods ins not sufficient.

File: operations/kubernetes/Kyverno.md (hunk 1 above)
Comment (yarlagadda-srinivasa-rao) [reply operations/kubernetes/Kyverno.md:444]: Hi @mtupitsyn, as discussed, it looks like this was synchronization issue, admission controller started before trust hook injects certificate. After restarting, things were working as expected. This was notices in recent installed systems. I will request Pradeep or Dinesh to attach the test logs here.

**Comment by yarlagadda-srinivasa-rao** [comment]
Hi @mtupitsyn, I am not able to review & approve this PR. Looks like I do not have permission. It looks like Kyverno has limitations with background scan. For 1.6.0, I am fine with this documentation changes. Thank you.

**Review by yarlagadda-srinivasa-rao — APPROVED** [review]

**Review by mtupitsyn — APPROVED** [review]

**Pull request merged**